src/libproxychains.c

```diff
@@ -858,5 +917,6 @@ DYLD_HOOK(freeaddrinfo);
 DYLD_HOOK(gethostbyaddr);
 DYLD_HOOK(getnameinfo);
 DYLD_HOOK(close);
+DYLD_HOOK(close_range);
```

Comment:
monterey hooking is only for OSX and afaik it doesn't have close_range

Reply:
Thanks for this knowledge! fixed in latest commit.